common/azure-core-test-perf/src/main/java/com/azure/core/test/perf/CountOptions.java

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.test.perf;
+
+import com.beust.jcommander.Parameter;
+
+/**
+ * Represents the count option for performance tests.
```

Comment:
A little bit more descriptive Javadoc will help users  of this framework. For example , explain what `count` mean here   and specify default if you have any.